reg_tests/global_cycle/driver.hera.sh

```diff
@@ -79,6 +79,12 @@ export COMOUT=$DATA
 TEST4=$(sbatch --parsable --ntasks-per-node=6 --nodes=1 -t 0:05:00 -A $PROJECT_CODE -q $QUEUE -J c48.noahmp.frac \
       -o $LOG_FILE -e $LOG_FILE ./C48.noahmp.fracgrid.sh)
 
+LOG_FILE=consistency.log05
+export DATA="${DATA_DIR}/test5"
+export COMOUT=$DATA
```

Comment:
I think this should be TEST5.

Reply:
And to trigger the 'summary' job, you will need to add ":$TEST5" before the "<< EOF"

Reply:
Thanks for catching the typo.